editor/plugins/editor_debugger_plugin.cpp

```diff
@@ -106,6 +104,26 @@ void EditorDebuggerSession::set_breakpoint(const String &p_path, int p_line, boo
 	debugger->set_breakpoint(p_path, p_line, p_enabled);
 }
 
+ScriptEditorDebugger::CameraOverride EditorDebuggerSession::get_camera_override() const {
+	ERR_FAIL_NULL_V_MSG(debugger, ScriptEditorDebugger::OVERRIDE_NONE, "Plugin is not attached to debugger.");
+	return debugger->get_camera_override();
+}
+
+void EditorDebuggerSession::set_camera_override(ScriptEditorDebugger::CameraOverride p_override) {
+	ERR_FAIL_NULL_MSG(debugger, "Plugin is not attached to debugger.");
+	debugger->set_camera_override(p_override);
+}
+
+const Tree *EditorDebuggerSession::get_editor_remote_tree() const {
+	ERR_FAIL_NULL_V_MSG(debugger, nullptr, "Plugin is not attached to debugger.");
+	return debugger->get_editor_remote_tree();
+}
+
+void EditorDebuggerSession::update_live_edit_root() {
+	ERR_FAIL_NULL_MSG(debugger, "Plugin is not attached to debugger.");
+	debugger->update_live_edit_root();
+}
+
```

Comment:
I don't think these should be in `EditorDebuggerSession`, which is just a class to expose debugger functionalities, to scripting.

Reply:
These are indeed functionalities of `ScriptEditorDebugger`. The game view is implemented as an `EditorDebuggerPlugin`, so I think `EditorDebuggerSession` is the intended way of accessing them.

These indirections can be worked around by calling `send_message()` directly. But I think we still need access to the corresponding `ScriptEditorDebugger` to keep its internal state in sync.

Maybe we can expose these? Or should I make the indirection via the `EditorDebuggerNode` singleton instead of via `EditorDebuggerSession`?